Documentation/Conditions/Index.rst

```diff
@@ -1141,6 +1141,10 @@ getTSFE
 
       [getTSFE().type == 98]
 
+   Current page id equals to 17. :ref:`PAGE`::
```

Comment:
Linking to the explanation of the `PAGE` TypoScript object does not add any value here IMO.

Reply:
https://docs.typo3.org/m/typo3/reference-typoscript/master/en-us/Setup/Page/Index.html

Only here you have an explanation for the id of the page.

Reply:
Before TYPO3 10 you could write the TypoScript Condition like this:

    [globalVar = TSFE:id = 279]

So I find this solution:

    [getTSFE().id == 279]

I could not find your solution at all.

Another hint could be added in order to refer to page based conditions.

Reply:
Exactly: https://docs.typo3.org/m/typo3/reference-typoscript/10.4/en-us/Conditions/Index.html#page

Reply:
I have added the local link to page instead.